posthog-node/package.json

```diff
@@ -28,8 +28,12 @@
   },
   "devDependencies": {
     "@types/node": "^18.0.0",
+    "@sentry/types": "^8.17.0",
     "commander": "^9.3.0"
   },
+  "peerDependencies": {
+    "@sentry/types": ">=8"
+  },
```

Comment:
Not all people would use this integration so its better to not set it as peer dependency but rather only `devDependencies`